coffeescript/rails.validations.simple_form.coffee

```diff
@@ -4,6 +4,9 @@
  * Licensed under MIT (http://opensource.org/licenses/mit-license.php)
 ###
 
+join_tag_and_classes = (tag, classes) ->
+  tag + '.' + classes.filter(Boolean).map((cls) -> cls.replace(/\ /g, '.')).join('.')
```

Comment:
I really like this approach but we cannot use this

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

map and filter requires IE9 but the current versions of CSV still supports IE8